wagtail/tests/test_comments.py

```diff
@@ -41,3 +43,34 @@ def test_deleting_most_recent_revision_deletes_created_comments(self):
         self.revision_3.delete()
         with self.assertRaises(Comment.DoesNotExist):
             self.new_comment.refresh_from_db()
+
+
+class TestContentPath(WagtailTestUtils, CommentTestingUtils, TestCase):
```

Comment:
This test doesn't seem to use `CommentTestingUtils`, so I think we can either remove it or refactor both. Removing it is easier so I'd go for that